load_testing/configs/1_200_ramp_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "locustfile_list": ["call_aaq.py"],
```

Comment:
Do i have to change this to test typebot instead? Could you add to the docs if so?

also, let's change the filenames to "aaq_search.py" and "typebot.py"?